python/foglamp/services/south/ingest.py

```diff
@@ -210,6 +210,13 @@ class Ingest(object):
 
         await cls._read_config()
 
+        # Auto adjust queue size in case plugin's 'dataPointsPerSec' value exceeds default queue size
+        try:
+            cls._readings_insert_batch_size = max(cls._readings_insert_batch_size,
+                                                  int(cls._parent_service.config['dataPointsPerSec']['value']))
```

Comment:
1. dataPointsPerSec is not a standard notation across all south plugins
2. We cannot have the batch size set to dataPointsPerSec on a low end device like raspberry pi. The Host machine will starve memory in that case. I suggest to keep it configurable

Reply:
@Singhal-Vaibhav  has a valid point here. We can not hard code this.

Reply:
Regarding first point, [FOGL-1780](https://scaledb.atlassian.net/browse/FOGL-1780) talks about standardisation of data points per second across all South plugins and Sinusoid plugin already implements this. So, IMHO, there is no reason to choose another name for this. 
Regarding second point, if you examine the code carefully, you will find that the batch size is there since beginning and is in play regardless of whether we are running FogLAMP on ubuntu or Raspberry. Secondly, this is a try/except loop. If a plugin does not have this item, it will simply be ignored and default/configured values would be choosen. Thirdly, it gives us a chance to auto adjust queue size which greatly benefits the readings buffer size sent to DB layer and helps in minimizing DB write lock frequencies and thus in increasing the throughput. And lastly, nothing stops us in choosing the queue size manually above the dps rate because choosing lower rate would choke the Ingest process that we have experienced so many times for high dps rates. Also, please note that currently queue size is not configurable per se and can be set directly in code only.